Allow Single Quotes for Pixel Jobs

### Is there an existing issue for this?

- [x] I confirm that I have not found an existing or similar issue.

### How to reproduce the issue

New job breaks old job page if we have a pixel that uses double quotes. For some reason we don't allow single quote pixels inside of the create new job panel but let them pass for the edit panel, let's consolidate the behavior to either let single quotes go through both times (preffered) or not allowing them for both

### Expected Result

We want to be able to use single quotations in pixels in the jobs tab

### Actual Results

When you use single quotations in a pixel on the new UI job page it errors out - see stack trace below
[INFO ] 2025-09-26 15:49:47 p.o.Insight:326 [user=mfreshwaters] Pixel >>> META|ScheduleJob(jobName=["testJob"], jobGroup=["defaultGroup"], cronExpression=["0 0 12 * * ? *"], cronTz=["US/Eastern"], recipe=["<encode>SetContext('29e9e371-9243-4293-ad3b-4be08ef95ab5')</encode>"], uiState='{"jobType":"Custom Job","jobName":"testJob", "cronExpression":"0 0 12 * * ? *","cronTimeZone":"US/Eastern","recipe":"SetContext('29e9e371-9243-4293-ad3b-4be08ef95ab5')","recipeParameters":""}',triggerOnLoad=[false],triggerNow=[false]); [ERROR] 2025-09-26 15:49:47 p.s.PixelRunner:98 [user=mfreshwaters] StackTrace:  prerna.sablecc2.parser.ParserException: [1,333] expecting: ',', ')'

### Additional Context

_No response_

### Requirements or Tasks

- [ ] Allow the UI to pass single quotes through when creating jobs

### Have you reproduced the issue with at least the latest released version?

- [x] Yes

### I have checked my terminal/console and have included relevant text here?

- [x] Yes

--- a/packages/client/src/pages/jobs/JobBuilderModal.tsx
+++ b/packages/client/src/pages/jobs/JobBuilderModal.tsx
@@ -252,9 +252,7 @@ export const JobBuilderModal = (props: {
 					builder.jobType
 				}","jobName":"${builder.name}", "cronExpression":"${
 					builder.cronExpression
-				}","cronTimeZone":"${builder.cronTz}","recipe":"${
-					builder.pixel
-				}","recipeParameters":""}',triggerOnLoad=[false],triggerNow=[false]);`,
+				}","cronTimeZone":"${builder.cronTz}", "recipeParameters":""}',triggerOnLoad=[false],triggerNow=[false]);`,
 			);
 			if (response.errors.length) {
 				throw new Error(response.errors[0]);

